scripts/pre_commit_linter.py

```diff
@@ -776,14 +776,21 @@ def _check_docstrings(all_files):
         filename for filename in all_files if not
         any(fnmatch.fnmatch(filename, pattern) for pattern in EXCLUDED_PATHS)
         and filename.endswith('.py')]
-    message = 'There should be a period at the end of the docstring.'
+    message_period = 'There should be a period at the end of the docstring.'
+    message_multi = 'Multiline docstring should end in a new line.'
+    message_single = ('Single line docstring should not span two lines. '
+        'If line length exceeds 81 characters, convert it to a multiline docstring.')
```

Comment:
Make the message variables a bit more descriptive. Something like: `message_for_period`, `message_for_multiline_docstring` and `message_for_single_line_docstring`. Also, we have an 80 characters limit and not 81. Please change it too.

Reply:
@apb7 Done!